docs/authorizer-guide/identity-context.mdx

```diff
@@ -8,26 +8,31 @@ description: The Identity Context and how it is used
 # Identity Context
 
 When an Authorizer evaluates a policy, it receives an Identity Context from the calling application.
-The `type` property of that Identity Context instructs the Authorizer how to identify the user:
```

Comment:
these are excellent updates. it would be great to also fix https://docs.aserto.com/docs/authorizer-guide/identity-context

Reply:
https://github.com/aserto-dev/aserto-docs/pull/203